src/frontend/src/layouts/AppLayout.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+  <div class="app-layout">
+    <IndexHome />
+  </div>
```

Comment:
Исходя из комментария по поводу хедера в файле Index.vue структура AppLayout получается примерно такая:
```suggestion
  <div class="app-layout">
    <AppLayoutHeader />
    <IndexHome />
  </div>
```